lib/Parser/rterrors.h

```diff
@@ -20,6 +20,7 @@
 #define RT_ERROR_MSG_UNUSED_ENTRY
 #endif
 
+// NOTE: The VBSERR macros should not be changed due to legacy dependencies outside of Chakra.
```

Comment:
Changing the naming is really the only issue here.

What should definitely not be changed is the definitions (numbers, messages, and so on). They are depended upon in a way which is definitely too risky to change.

Other projects with a dependency on the actual names as they are is less of a concern (since this file itself is apparently only referenced within Chakra), but it would be nice to keep the names as they are for quality of life when they go looking through the code (including our code) for instances of those macros named as they are.